docs/DeveloperGuide.adoc

```diff
@@ -137,111 +137,348 @@ image:BetterModelClassDiagram.png[]
 .Structure of the Storage Component
 image::StorageClassDiagram.png[]
 
-*API* : link:{repoURL}/src/main/java/seedu/address/storage/Storage.java[`Storage.java`]
+*API* : link:{repoURL}/src/main/java/io/xpire/storage/Storage.java[`Storage.java`]
 
 The `Storage` component,
 
 * can save `UserPref` objects in json format and read it back.
-* can save the Address Book data in json format and read it back.
+* can save the Xpire data in json format and read it back.
 
 [[Design-Commons]]
 === Common classes
 
-Classes used by multiple components are in the `seedu.addressbook.commons` package.
+Classes used by multiple components are in the `io.xpire.commons` package.
 
 == Implementation
 
 This section describes some noteworthy details on how certain features are implemented.
 
+// tag:tobuylist[]
+
+=== [Proposed] Replenish List
+==== Implementation
+//{Explain here how the ToReplenish/ToBuy List is implemented}_
+Items are added to the replenish list by the user using the replenish command.
+When an item expires, the item is automatically tagged as "expired".
+This enables the user to search for a list of expired items by searching for the "expired" tag.
+With this, the user is able to view the list of expired items and decide which items to add to the replenish list.
+On the other hand, when an item has run out, the user is prompted to replenish the item using the replenish command.
```

Comment:
Maybe can consider giving a brief description of what replenish is used for before talking about its implementation